docs/README.es.md

```diff
@@ -20,7 +20,7 @@ Pingvin Share es una plataforma de intercambio de archivos autoalojada y una alt
 ## 🐧 Conoce Pingvin Share
 
 - [Demo](https://pingvin-share.dev.eliasschneider.com)
-- [Reseña por DB Tech](https://www.youtube.com/watch?v=rWwNeZCOPJA)
+- [Reseña realizada por No Solo Hacking](https://www.youtube.com/watch?v=ocd4EpLTYkU)
```

Comment:
```suggestion
- [Reseña realizada por No Solo Hacking (español)](https://www.youtube.com/watch?v=ocd4EpLTYkU)
- [Reseña por DB Tech (inglés)](https://www.youtube.com/watch?v=rWwNeZCOPJA)
```

Great thanks for your video :) I would keep the review of DB Tech in the README but add a note that this review is on English. Above is my suggestion but I'm not sure if it is grammatically correct.